src/functions.js

```diff
@@ -58,14 +58,17 @@ var ParseError = require("./ParseError");
  *
  * The last argument is that implementation, the handler for the function(s).
  * It is called to handle these functions and their arguments.
- * Its own arguments are:
- *  - func: the text of the function
- *  - [args]: the next arguments are the arguments to the function,
- *            of which there are numArgs of them
+ * It receives two arguments:
+ *  - context contains information and references provided by the parser
+ *  - args is an array of arguments obtained from TeX input
+ * The context contains the following properties:
+ *  - func: the text (i.e. name) of the function, including \
```

Comment:
This should be "funcName" so that it's consistent with the other uses of "func" and "funcName".